tmva/sofie/inc/TMVA/ROperator_Shape.hxx

```diff
@@ -0,0 +1,83 @@
+#ifndef TMVA_SOFIE_ROPERATOR_Shape
+#define TMVA_SOFIE_ROPERATOR_Shape
+
+#include "TMVA/SOFIE_common.hxx"
+#include "TMVA/ROperator.hxx"
+#include "TMVA/RModel.hxx"
+
+#include <sstream>
+#include <iostream>
+#include<sstream>
+#include<vector>
+#include <iterator>
+#include<string>
+namespace TMVA{
+namespace Experimental{
+namespace SOFIE{
+
+template <typename T>
+class ROperator_Shape final : public ROperator
+{
+
+private:
+
+   /* Attributes*/
+   std::string fNX;
+   std::string fNY;
+   std::string result;
```

Comment:
Why we have this result string stored ? See comments below.
One should also follow coding conventions with data member starting with `f` and have capitalize second letter.